SPECS/ksh/ksh.spec

```diff
@@ -80,12 +73,12 @@ touch %{buildroot}%{_bindir}/rksh
 touch %{buildroot}%{_mandir}/man1/rksh.1.gz
 
 %check
-# We run more tests as non-root user
-chmod g+w . -R
-useradd test -G root -m
-
-# Disabling tests as they tend to freez and the package is low-pri at the moment.
-false && sudo -u test ./bin/shtests --compile
```

Comment:
Can we still run the test as a non-root user? Or are all tests now running even if it's from a root account?